src/chapter3/bash.md

```diff
@@ -1 +1,40 @@
 # Bash Scripts
+
+Bash is both a command line interface and a scripting language. Bash scripts are a series of commands that are executed in order. Bash scripts are useful for automating tasks that you do often, or for running a series of commands that you don't want to type out every time. In our case, Bash scripts are used for running jobs on M3.
```

Comment:
Maybe say something like 'commands on Linux (generally) us Bash' to 'here's what Bash is more generally ...' to 'you can encapsulate any commands into a script that can be easily reused'. Maybe also mention how any command you would run normally can be put in a Bash script.